vlm/src/main/scala/geotrellis/contrib/vlm/gdal/GDALBaseRasterSource.scala

```diff
@@ -6,11 +6,14 @@ import geotrellis.raster._
 import geotrellis.raster.reproject.Reproject
 import geotrellis.raster.resample.ResampleMethod
 import geotrellis.vector._
+
 import org.gdal.gdal.{Dataset, gdal}
 import org.gdal.osr.SpatialReference
 
 trait GDALBaseRasterSource extends RasterSource {
+  val uri: String
```

Comment:
Why did you add `uri` here as a `val` when `RasterSource` already has it as a method?